app/controllers/sessions_controller.rb

```diff
@@ -1,4 +1,9 @@
 class SessionsController < ApplicationController
+
+  before_action :redirect_https, only: [
+    :login
```

Comment:
I think it makes sense to always redirect to https, not just for login. It's the more privacy-respecting option. Let's remove the `only` clause from here, @akp2603

Reply:
That would be a much bigger change. I am a little new to this adding ssl redirection. IMO, let's just do it for this action this time and if it works perfectly, we can add the before action in the application controller only. (That's actually the reason I kept in application_controller. Even I think this should be on all the actions. But let's just try it out with this one first. Thoughts?)

Reply:
That's fair. I'm fine leaving it small for now.